packages/suite/src/components/suite/Coin/index.tsx

```diff
@@ -27,6 +30,12 @@ const SettingsWrapper = styled.div<{ onClick: ((e: React.MouseEvent) => void) |
                 )};
             }
         `}
+
+    ${props =>
+        !props.toggled &&
+        css`
+            pointer-events: none;
```

Comment:
there is one more issue - on touch device if you click on the right side of the **active** coin "badge" it will open the settings instead of deactivating the coin

Reply:
Oh, you're right.

Reply:
on it

Reply:
a8cbcc66f1791e0907ebd56df69e999129302f4d ?

Reply:
I'm not sure about the `and (pointer: coarse)` part, why not only `@media (hover: none)`?

Reply:
It's already done.